src/groupeddataframe/grouping.jl

```diff
@@ -50,14 +50,17 @@ groupby(cols; sort = false, skipmissing = false)
 
 An iterator over a `GroupedDataFrame` returns a `SubDataFrame` view
 for each grouping into `d`. A `GroupedDataFrame` also supports
-indexing by groups and `map`.
+indexing by groups, `map` (which applies a function to each group)
+and `combine` (which applies a function to each group
+and combines the result into a data frame).
```

Comment:
Can `combine` return something else than `DataFrame`?

Reply:
This is what I understand - so we should write `DataFrame` not `data frame` right?

Reply:
I'm not sure it really matters. It wouldn't be unreasonable for a custom `AbstractDataFrame` type to return something other than a `DataFrame`, even if currently that's not the case.